src/inputs/input.jsx

```diff
@@ -151,40 +201,36 @@ class Input extends React.PureComponent {
                 }
 
                 this.inputTimer = setTimeout(() => {
-                    if (_.isEmpty(newValue)) {
+                    if (isEmpty(newValue)) {
                         if (required) {
-                            newValue = _.isNumber(min) ? min : (_.isNumber(max)? max : 0);
+                            if (isNumber(min)) {
+                                newValue = min;
+                            } else if (isNumber(max)) {
+                                newValue = max;
+                            } else {
+                                newValue = 0;
+                            }
                         }
                     } else {
-                        newValue = +newValue;
-                        if (_.isNumber(max)) {
+                        newValue += 1;
```

Comment:
Why `+= 1` here?
This looks a bit suspect.
The replaced code seems to have been `newValue = +newValue;`, using the unary `+` operator to "forcibly cast" strings to numbers I think.

Reply:
I have tested these lines and they are working. The above also appeases Eslint's styling. Are you thinking there could be a potential regression that I am not thinking about?

Reply:
Yeah, isn't it possible that "old" value was, say, 5, and new value is now, say, 10?  Incrementing or decrementing by 1 is not the only possible change ... right?

Reply:
Fixed. Thanks for pointing that out and explaining what I was not seeing... I was seeing `++` for whatever reason.

Reply:
The very reason we do code reviews! :-)  Get multiple people's eyes on.  All good.  Happens to the best of us.
Looks like we restored the prior code; resolving.